crates/symbolicator/src/services/cficaches.rs

```diff
@@ -87,10 +87,9 @@ impl CfiCacheActor {
 
 #[derive(Debug)]
 pub struct CfiCacheFile {
-    object_type: ObjectType,
-    identifier: ObjectId,
-    scope: Scope,
-    data: ByteView<'static>,
+    // NOTE: while this really is a dead store, we keep it since the `ByteView` should keep the
+    // underlying file alive via its mmap mapping.
+    _data: ByteView<'static>,
```

Comment:
Is this assumption right? Even if so, linux still allows `unlink`, so even if the file itself is still there (in limbo), the procspawn-breakpad sub-process could still fail to open that file, right?